debian-setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Installs all of the necessary packages in a Docker container
+# MUST BE RUN WITH ROOT
```

Comment:
Does the above `script` section of `.travis.yml` need to run this as root?

Reply:
`docker`'s default user is root. Kind of a superfluous warning, I can remove it if necessary.

Reply:
Is that important when running docker locally?

Reply:
Don't think so since it should be the same for local builds:

```
$ docker run --rm -ti debian
root@8b6e6bd76cbd:/# 
```

Reply:
ok, please remove the comment then.  The caps makes it seems like there's an action item, but I don't see sudo being used in this file, or in the travis.yml, or locally.